lab_1/main.py

```diff
@@ -74,27 +67,17 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list:
     top_n = 1
     --> ['happy']
     """
-    common_words = []
-    n = 0
-
     sorted_dict = dict(sorted(freq_dict.items(), key=lambda item: item[1], reverse=True))
     sorted_dict = list(sorted_dict.keys())
 
-    if top_n > len(sorted_dict):
-        top_n = len(sorted_dict)
     if top_n >= 0:
```

Comment:
can you move this check to the beginning? 
if top_n is incorrect, there is no point doing anything at all